Simplified/NYPLBookDetailTableView.swift

```diff
@@ -105,7 +105,7 @@ final class NYPLBookDetailTableViewDelegate: NSObject, UITableViewDataSource, UI
     refresh()
   }
   
-  func updateFonts() {
+  @objc func updateFonts() {
```

Comment:
Same as above, I'm suspicious if `@objc` is needed if the entire class is marked `@objcMembers` Just let me know if it works after removing these on the functions.